### ISSUE TITLE
Transfer mongoid-autoinc to github/mongoid?

### ISSUE THREAD

**Issue by dblock** [suweller/mongoid-autoinc#31]
I had https://github.com/mongoid kindly donated to the community by the nice folks at MongoDB. The three people who have org level admin access are currently @durran, @estolfo and myself. 

I want to invite this project to be moved to the org. I think it makes good sense to group mongoid projects together and have continuous support from MongoDB, the company, over the long term. All admins of this project will continue having admin access of course, but this would reduce the bus factor since currently only @suweller can add maintainers here. 

If you think it's a good idea, please transfer this project to me and I'll move it into the org and please add `dblock@dblock.org`, `emily@mongodb.com` and `durran@gmail.com` to rubygems.

**Comment by suweller** [comment]
It looks like that has changed already from three people involved to a single user. Do you still want me to consider moving `mongoid-autoinc` to the `mongoid` organisation?

**Comment by dblock** [comment]
Yes, because if you decide to give up @suweller there's no way anyone can reclaim this repo and its administration. Just trying to help reduce potential bus factor.

**Comment by niightly** [comment]
any news on this?

**Comment by romansavrulin** [comment]
@dblock @suweller also wondering if there any news on this

**Comment by dblock** [comment]
I am around if I can be of any help!

**Comment by romansavrulin** [comment]
@dblock Can you please answer what's the best way to implement atomic auto increment operations that can be safely run on a heavily loaded cluster with several rails web app instances behind load balancer? I saw [some implementations](https://www.tutorialspoint.com/mongodb/mongodb_autoincrement_sequence.htm) with javascript functions, but this seems to be not available in mongoid. Any ideas?

**Comment by dblock** [comment]
I don't know the answer to this @romansavrulin, sorry. This gem seems like what you need, but YMMV

**Comment by niightly** [comment]
Hey people! 
just passing by to see if there is any update on this? I mean, it will be integrated within mongoid, or did you change your mind?

**Comment by suweller** [comment]
@johnnyshields @durran Mongoid-Autoinc is not getting the attention it deserves. I love this project as it reminds me of my early days starting out as an enthusiastic ruby developer. Currently, I'm running my own team, have people responsibilities inside and outside work, as well as teaching weekly dance classes. Because of this... with some pain in my heart... I think it is better to migrate it and host it amongst the other mongoid repos and gems. How would we go about doing this?
